### DISCUSSION TITLE
Print Tax Summary with more than one taxes using browser print

### DISCUSSION THREAD

**Discussion by mbugikp** [Blair2004/NexoPOS#1451] [Ideas]
**Describe the bug**
It is illegal in our country not to show applicable taxes on the receipt, The law requires to show detailed description and amount of each tax on a receipt. The current browser print just summaries tax & gives one figure when you have applied more than one tax on an item.

**To Reproduce**

**Expected behavior**
All receipts / invoices should show Details of applicable taxes and amounts irrespective of how they are printed

**Screenshots**

**Environment:**
 - NexoPOS 4.x vesrion: [e.g. 4.5.2]
 - Premium Extensions: [e.g Nexo Print Adapter (4.5.0), Gastro (4.5.2)]

**Comment by mbugikp** [comment]
Thank you, much appreciated

**Comment by Blair2004** [comment]
Added on v4.8.17. If you're using NPS Adapter, you'll also need to update to 4.8.7

**Comment by mbugikp** [comment]
Thank you @Blair2004
